src/test/java/core/basesyntax/enums/OperationTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.enums;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class OperationTest {
+
+    public static final String[] VALID_CODES = {"b", "s", "p", "r"};
+    public static final Operation[] EXPECTED_OPERATIONS = {
+            Operation.BALANCE, Operation.SUPPLY, Operation.PURCHASE, Operation.RETURN};
```

Comment:
What if you add a new operation? You can write methods to get all operations and operation codes in Operation.

Reply:
Thank you very much! I just changed it to:
`public static final Operation[] EXPECTED_OPERATIONS = Operation.values();`
To get all operations.